src/Applications/LDAS_App/process_hist.csh

```diff
@@ -53,7 +53,7 @@ endif
 
 if($NENS > 1) then
    set GridComp = ENSAVG
-   sed -i 's|VEGDYN|'VEGDYN0000'|g' $HISTRC
+   sed -i 's|VEGDYN|'VEGDYN_e0000'|g' $HISTRC
```

Comment:
I think this change should not be on this PR and belongs into #584 .  The present PR goes directly to "develop".

Reply:
Oh no, no idea why this change shows up here. You are right, it should be in other PR